libraries/helpers.rb

```diff
@@ -447,6 +448,7 @@ def share_munge_head_node
       # Copy key to shared dir
       cp /etc/munge/munge.key /home/#{node['cluster']['cluster_user']}/.munge/.munge.key
     HEAD_SHARE_MUNGE_KEY
+    creates "/home/#{node['cluster']['cluster_user']}/.munge/.munge.key"
```

Comment:
same comment here. why wouldn't we re-generate it if a key already existed?

Reply:
We do not want to overwrite data under /home directory, in general.
Do you agree?